sdks/js/src/types.ts

```diff
@@ -463,7 +463,7 @@ type BrowseActionPublicType = z.infer<typeof BrowseActionTypeSchema>;
 const ConversationFileTypeSchema = z.object({
   fileId: z.string(),
   title: z.string(),
-  contentType: z.string(),
+  contentType: SupportedContentFragmentTypeSchema,
```

Comment:
I think it means an old client will break if we send back a new format that wasn't supported before. (we check that the response match the schema)

Reply:
This is a new type that was introduced in the context of that project. Not used yet

Reply:
I mean "old clients" in the sense that client that will use Dust-Client vX and post vX, we can return a new content-type from the api.

But I checked and `SupportedContentFragmentTypeSchema` uses `FlexibleEnumSchema` that I introduced recenty and avoid this issue.